### ISSUE TITLE
An odd "phenomenon" while usage MetroWindow and Canvas.

### ISSUE THREAD

**Issue by DeputyOfCopyPaster** [MahApps/MahApps.Metro#3638]
Hello everyone!
**Metro 1.6.5**
I've noticed that something wrong happens with images (Canvas) while Metro framework usage.
I performed two simpliest tests and I had been surprised that  pure WPF project is deprived of such a "phenomenon" as follows:

![polygons](https://user-images.githubusercontent.com/43616343/66382174-62621380-e9c3-11e9-9265-596cd8a0ca33.png)
![polygons-gif](https://user-images.githubusercontent.com/43616343/66382381-c5ec4100-e9c3-11e9-9cac-bffb32146194.gif)

In other words the pure WPF project handles well with all Canvases inserted.
So, my question is, how to get rid of that "phenomenon"?

Please any insights on that issue?

thanks in advance!

**An Update:** the same disaster has v.2.00-alpha0490.

**Comment by timunie** [comment]
@DeputyOfCopyPaster can you upload your sample? I have an idea, but I have to try it tomorrow if it works. 

Happy coding 
Tim

**Comment by timunie** [comment]
@DeputyOfCopyPaster Please have a look here: [https://stackoverflow.com/questions/2399731/when-should-i-use-snapstodevicepixels-in-wpf-4-0](https://stackoverflow.com/questions/2399731/when-should-i-use-snapstodevicepixels-in-wpf-4-0)

**Comment by DeputyOfCopyPaster** [comment]
Tim, 
ok, please look at the sample attached. To see difference look firsrt at pure WPF project, then add MetroWindow by yourself and catch the "phenomenon".





[CanvasPhenomenon.zip](https://github.com/MahApps/MahApps.Metro/files/3702046/CanvasPhenomenon.zip)

**Comment by DeputyOfCopyPaster** [comment]
> 
> 
> @DeputyOfCopyPaster Please have a look here: https://stackoverflow.com/questions/2399731/when-should-i-use-snapstodevicepixels-in-wpf-4-0

**Comment by punker76** [comment]
@DeputyOfCopyPaster The SnapsToDevicePixels of the MetroWindow is set to true, so all inner controls inherits this value. You can fix this by changing the SnapsToDevicePixels for the Canvas and set it to false.

```xaml
<Canvas SnapsToDevicePixels="False">
</Canvas>
```

**Comment by DeputyOfCopyPaster** [comment]
**An Update:**

@timunie, @punker76 
your link above, led to the assumption that it might make a sense.
As it turned out, I 've gotten better Canvas' behaviour as it was before. At least the lines stopped disappearing. :+1:
